src/webvr-manager.js

```diff
@@ -98,9 +98,10 @@ function WebVRManager(renderer, effect, params) {
       this.isVRCompatible = true;
     } else if (hmd) {
       this.isVRCompatible = true;
+      this.usingPolyfill = hmd.deviceName.indexOf('webvr-polyfill') === 0;
       // Only enable distortion if we are dealing using the polyfill, we have a
```

Comment:
Oh, looks like that hasn't been released. Se maybe we can merge this now and update it when polyfill is updated.